RosNet.Test/ApiTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Xunit;
+using System.IO;
+using RosNet.DataModel;
+using RosNet.RosReader;
+using System.Collections.Generic;
+using RosNet.Field;
+
+namespace RosNet.Test
+{
+    public class ApiTests
+    {
+        [Fact]
+        public void TestGetConnectionFields()
+        {
+            string path = Directory.GetCurrentDirectory() + "/TestBags/acc19-11.bag";
```

Comment:
Even without a bag this is not what you want, look at e.g. https://stackoverflow.com/a/68258659